### DISCUSSION TITLE
Wordpress REST API encountered an error - cURL error 7

### DISCUSSION THREAD

**Discussion by june415** [EasyEngine/easyengine#1513] [Support Requests]
Hi,

Recently I installed EE4 on the new OS and move the existing wordpress websites to the new server. Everything seems works well except wordpress REST API doesn't work.

The error message as below:
The REST API is one way WordPress, and other applications, communicate with the server. One example is the block editor screen, which relies on this to display, and save, your posts and pages.
The REST API request failed due to an error.
Error: cURL error 7: failed to connect to myexample.site port 443: Connection refused(http_request_failed).

The port 80 & 443 is available and I checked firewall seems no issue as well.

Then I spent a lot time to google and found below solution to fix the issue.

Add extra_hosts at docker-compose file.
extra_hosts:
    - myexample.site: new_server_ip
    
However, I'm not able to access /ee-admin tool after added extra_hosts. I receive "403 Forbidden". Before that I had no issue to visit.

I think there is something to do with docker-compose. But I'm totally new with docker.

Can anyone advise me how i should configure the docker setting?

Many thanks!

**Comment by robbie-berns** [comment]
Same problem on all my sites with Ubuntu 18.04 and EE. Watching

**Comment by robbie-berns** [comment]
I actually found the answer in an earlier discussion here. Add to /etc/hosts:

site.ip.add.ress example.com

Then run `ee site restart example.com`
Fixed

**Reply by june415** [reply]
My /etc/hosts folder is empty....do you have access issue to ee-admin after add the above code?

**Reply by robbie-berns** [reply]
/etc/hosts is an ascii file not a folder (easy to mistake in /etc). It is there by default in all installations.

And no, the ee-admin still works.

**Reply by prashanthpss** [reply]
It worked !! Thanks

**Reply by awps** [reply]
Thanks. I tried to add it to hosts as is but later realized that it's: `MY_LOCAL_IP_ADDRESS example.com` :D

**Comment by marcoandrei** [comment]
Hello, @june415! 

Have you resolved your problem?

These were the steps I took to resolve mine:

1) Edit the file /etc/hosts:

`vi /etc/hosts`

2) Add the following line at the end of the document (substitute 200.248.206.10 by your own server IP):

`200.248.206.10   mydomain.com`

3) Restart the site:

`ee site restart mydomain.com`

That is it! I hope I could help you.

Saúde!
